src/models/core/AnswersConfig.ts

```diff
@@ -65,9 +58,7 @@ export interface AnswersConfigWithApiKey extends BaseAnswersConfig {
  * @public
  */
 export interface AnswersConfigWithToken extends BaseAnswersConfig {
-  /**
-   * {@inheritDoc BaseAnswersConfig.token}
-   */
+  /** The authentication token of the answers experience. */
```

Comment:
Maybe we should add to this so that it reads: "The authentication token of the answers experience which will be passed in the Auth header as a Bearer token"